docs/pywbemcli/cmdlineinterface.rst

```diff
@@ -232,7 +232,7 @@ list of the supported command groups and commands without command group.
 
     $ pywbemcli
     pywbemcli> --help
-    Usage: pywbemcli [GENERAL-OPTIONS] COMMAND [ARGS]...
+    Usage: pywbemcli [GENERAL-OPTIONS] COMMAND [ARGS] [CMD-OPTIONS]
     . . .
 
     Options:
```

Comment:
We currently have a section heading "Options:" in the help text that is ambiguous as to whether it lists the `GENERAL-OPTIONS` or the `CMD-OPTIONS`. If technically possible, I think it would be great if that section heading could be changed to "General options:" for the top level help, and to "Command options:" for the lower level help.
PS: I'm just adding this here because there is an "Options:" section shown.
 Suggestion:
* Merge `[COMMAND-GROUP] COMMAND` in usage to just `COMMAND` which is consistent with the section title "Commands:".
* Change title of "Options:" section to "General options:".

Reply:
**Discussion point:** How do we go about this?

Reply:
I have described that discussion point in more detail in issue #446.